test/cri-metrics.bats

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bats
+# vim: set syntax=sh:
+
+load helpers
+
+function setup() {
+    setup_test
+    PORT="9090"
+    CONTAINER_ENABLE_METRICS="true" setup_crio
+    cat << EOF > "$CRIO_CONFIG"
+[crio.stats]
+collection_period = 0
+included_pod_metrics = [
+    "network",
```

Comment:
@haircommander should we have processes here also?

Reply:
yeah I think so

Reply:
it should be yeah. but they are not yet supported in the collection implementation.